test/CMakeLists.txt

```diff
@@ -3,10 +3,11 @@ include(GoogleTest)
 # Needed for creating the `testUil`-library.
 add_subdirectory(util)
 
-# Link binary ${basename} against `gmock_main`, the threading library,
-# and all libraries that are specified as additional arguments.
+# Link binary ${basename} against `gmock_main`, the threading library, the
+# general test utilities and all libraries that are specified as additional
+# arguments.
 function (linkTest basename)
-    qlever_target_link_libraries(${basename} ${ARGN} gmock_main ${CMAKE_THREAD_LIBS_INIT})
+    qlever_target_link_libraries(${basename} ${ARGN} gmock_main testUtil ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
You also have to add the `testUtil` library in line 43 for the `SingleTestBinary` case.
That is the reason for the failing Codecov build.

Reply:
Ah, my bad. I didn't think, that the infrastructure wouldn't always use the linking function, so I overlooked it. Fixed in [f35d2de](https://github.com/ad-freiburg/qlever/pull/1016/commits/f35d2dea5feccf59a7d70bd10dc96d14bd50ac20).